Documentation/Blazorise.Docs/Pages/Docs/Components/Radios/RadioPage.razor

```diff
@@ -42,6 +42,9 @@
         <BasicRadioGroupExample />
     </DocsPageSectionContent>
     <DocsPageSectionSource Code="BasicRadioGroupExample" />
+    <DocsPageParagraph>
```

Comment:
DocsPageParagraph should not be placed here.

Do it similar to Accordion example https://github.com/Megabit/Blazorise/blob/3912d18bd1329f4d2531e29d5c88613bb41b99b4/Documentation/Blazorise.Docs/Pages/Docs/Components/Accordions/AccordionPage.razor#L69